internal/zinc-compile-core/src/main/java/xsbti/compile/ClasspathOptionsUtil.java

```diff
@@ -34,6 +33,16 @@ public static ClasspathOptions boot() {
         return ClasspathOptions.of(true, false, false, true, true);
     }
 
+    /**
+     * Define {@link ClasspathOptions} where the Scala standard library is present in the classpath.
+     */
+    public static ClasspathOptions noboot(String scalaVersion) {
+        if (scalaVersion.startsWith("3.") || scalaVersion.startsWith("2.13."))
```

Comment:
For Scala 4 preparedness:

```suggestion
        if (!scalaVersion.startsWith("2.") || scalaVersion.startsWith("2.13."))
```